### PR TITLE
chore: update orange icons to accessible color

### PR DESCRIPTION
**Pull request by ErikSin**
Updates the add button and sync button to use an accessbile color (also updates both of them to use SVGs).

There also was a duplicate export of the `SyncIcon` so I deleted the file, and updated the import to come from icons/index

fixes (partially) #768

### CODE REVIEW

File: src/frontend/sharedComponents/AddButton.tsx [hunk 1]
```diff
@@ -27,16 +26,9 @@ const AddButtonNoMemo = ({
...
     <TouchableOpacity disabled={disabled || isLoading} onPress={onPress}>
-      {!isLoading ? (
-        <Image
-          source={require('../images/add-button.png')}
-          style={styles.button}
-        />
-      ) : (
-        <Loading size={15} />
-      )}
+      {isLoading ? <UIActivityIndicator size={50} /> : <AddButtonSVG />}
```
Comment (ErikSin) [inline src/frontend/sharedComponents/AddButton.tsx:31]: I also updated this loader. This was a legacy thing before we were using UiActivityIndicator. Updating it to be consistent with the rest of the app

Comment (cimigree) [inline src/frontend/sharedComponents/AddButton.tsx:31]: Can you remove the button styles as they are no longer being used?

File: src/frontend/sharedComponents/HomeHeader.tsx [hunk 2]
```diff
@@ -25,7 +25,7 @@ export const HomeHeader: FC<
         onPress={() => {
           navigation.navigate('Sync');
         }}>
-        <SyncIconCircle />
+        <SyncIconCircle testID="MAIN.sync-icon" />
```
Comment (cimigree) [inline src/frontend/sharedComponents/HomeHeader.tsx:28]: Thanks for adding the test id. It is working!

**Review by cimigree — APPROVED** [review]
Wow that is a bright orange! I like it!
Just one bit of code to remove please and then you can merge. All looked good and seemed to be working.

**Pull request merged**